lib/msal-browser/src/interaction_client/BaseInteractionClient.ts

```diff
@@ -75,6 +76,7 @@ export abstract class BaseInteractionClient {
             this.correlationId
         );
         this.performanceClient = performanceClient;
+        this.requestRetried = false;
```

Comment:
Thanks for making the change. Are we going to retry for interactive+silent calls? For silent, I think it makes sense to embed this with the `silentIframeClient`, since that is the only path for /authorize call.

Reply:
Yes, retry for silent will be in PR #7218 , redirect will be in another.